lib/booker/client.rb

```diff
@@ -24,9 +24,11 @@ class Client
     ENV_BASE_URL_KEY = 'BOOKER_API_BASE_URL'.freeze
     DEFAULT_BASE_URL = 'https://api-staging.booker.com'.freeze
     DEFAULT_AUTH_BASE_URL = 'https://api-staging.booker.com'
+    DEFAULT_REQUEST_TIMEOUT = 60
 
     def initialize(options = {})
       options.each { |key, value| send(:"#{key}=", value) }
+      self.request_timeout ||= DEFAULT_REQUEST_TIMEOUT
```

Comment:
Looks fine although in the future we may want to expose this as configuration via ENV, if it becomes something we need to adjust in the future more easily.